### DISCUSSION TITLE
Maximize window without going full screen. (Don't cover taskbar on Windows OS)

### DISCUSSION THREAD

**Discussion by xpufx** [wezterm/wezterm#1688] [Q&A] [answered]
Is there a shortcut to maximize a window without a title bar? Full screen (ALT+ENTER) makes the app full screen, covering the taskbar.

**Comment by wez** [comment, marked as answer]
Not in wezterm itself, but you can use `WIN+UpArrow` to maximize and `WIN+DownArrow` to restore in any app on Windows.
You might need to be on the latest nightly for this to work with all variations of `window_decorations`.

**Reply by directormac** [reply]
Ahhh didn't even know this existed on windows. LOL i was gonna make a thread about toggling maximization just like full screen.

**Comment by ardrigh** [comment]
Just noting in GNOME this appears to work with the same keys win/cmd+uparrow to maximize without using windows decorations.

**Comment by mendhak** [comment]
Alt+F10 should work as well, tested on Mint. This can be useful in case your Super+Up starts tiling the window
